Detect devices for brands: Asus, Oppo, Huawei, Sony, Lenovo, LG, Wiko, Nuvo, Alcatel, Irbis, Heair, Motorola, CoolPad, Sunvell, AllDocube,  Assistant, DEXP, ZTE, CAT,  Xiaomi, Google, Yandex, Hisense, T-Mobile, Allview, ViewSonic, Yxtel, Ginzzu, HTC

sanchezzzhak opened the pull request.

sanchezzzhak committed:
feat (device) Oppo Find X Lamborghini (PAHM00), R17 Pro (PBDM00)

issue #6174

sanchezzzhak committed:
feat (device) Oppo Reno (PCAM00, PCAT00), Reno Z (PCDM10)

issue #6174

sanchezzzhak committed:
feat (device) Oppo Reno K3 (PCGM00), A11 (PCHM10), A9 (2020) (PCHM30)

issue #6174

sanchezzzhak committed:
feat (device) Oppo Reno 2 (PCKM00), Reno 2Z (PCKM80), Reno Ace (PCLM10), A8 (PDBM00)

issue #6174

sanchezzzhak committed:
feat (device) Huawei P Smart (2019) (POT-LX2J)

sanchezzzhak committed:
feat (device) Oppo Realme 5S (RMX1925)

sanchezzzhak committed:
feat (device) Oppo Realme 5 (RMX1927)

sanchezzzhak committed:
feat (device) Oppo Realme X2 Pro (RMX1931)

sanchezzzhak committed:
feat (device) Oppo Realme X2 (RMX1991), Realme C2 (RMX1942)

sanchezzzhak committed:
feat (device) Oppo Realme X2 (RMX1992), Realme X2 Dual (RMX1993)

sanchezzzhak committed:
feat (device) Huawei Nova 5 (SEA-AL00)

sanchezzzhak committed:
feat (device) Huawei MediaPad M5 Pro 5 (SCM-AL09), MediaPad M6 10 (SCM-W09)
feat (device) Sony Xperia XZ3 (SO-01L)

sanchezzzhak committed:
feat (device) Wiko Y60 (W-K510-EEA)

sanchezzzhak committed:
feat (device) Asus ROG Phone 2 (ZS660KL)

sanchezzzhak committed:
feat (device) Alcatel 1S (5024D_RU)

sanchezzzhak committed:
feat (device) Nuvo NS35, ND40, ND45

sanchezzzhak committed:
feat (device) Xperia A2 (SO-04F)

sanchezzzhak committed:
feat (device) Irbis TZ195,TZ198, TZ742, TZ752, TZ885, TZ965, TZ969

sanchezzzhak committed:
feat (device) Haier L7 (HM-N700-FL)

sanchezzzhak committed:
feat (device) Motorola Moto Z2 Play (XT1710-02)

sanchezzzhak committed:
feat (device) Motorola Moto G5s (XT1799-2)

sanchezzzhak committed:
feat (device) Coolpad Cool Play 6 (VCR-A0)

sanchezzzhak committed:
feat (device) add new brand Sunvell && detect tv box model - T95Z Plus

sanchezzzhak committed:
feat (device) Alcatel improvement detect model 5099D

sanchezzzhak committed:
feat (device) Assistant Cetus (AT-108)

sanchezzzhak committed:
feat (device) add new brand AllDocube && detect tablet model - M5XS (T1001XS)

sanchezzzhak committed:
feat (device) DEXP (AL140, AL240)

sanchezzzhak committed:
feat (device) Huawei Mate 30 Pro (LIO-AN00)

sanchezzzhak committed:
feat (device) CAT B35

sanchezzzhak committed:
Merge branch 'master' into pr_device_32

sanchezzzhak committed:
feat (device) Sharp Aquos R2 (SH-M09), Aquos S2 (SH-M08)

sanchezzzhak committed:
feat (device) add new brand Ugoos AM6 (smart tv)

sanchezzzhak committed:
feat (device) Vivo Z1i (V1730DT), Z3x (V1730GA), Z1 (V1801A0), X23 (V1809A)

sanchezzzhak committed:
feat (device) Black Fox - B8 Fox (BMM441S), B4 (BMM543S)

sanchezzzhak committed:
feat (device) Asus ROG Phone 2 (I001D, I001DB, I001DC, I001DE)

sanchezzzhak committed:
feat (device) Asus ZenFone 6 (I01WD, I01WDX)

sanchezzzhak committed:
feat (device) ZenFone Max M2 (X01BD)

sanchezzzhak committed:
feat (device) Xiaomi Black Shark Helo (AWM-A0);
feat (device) Xiaomi Black Shark 2 Pro (DLT-A0);

sanchezzzhak committed:
feat (device) Huawei P30 (ELE-TL00);

sanchezzzhak committed:
feat (device) Huawei Mate 20 X (EVR-N29);

sanchezzzhak committed:
feat (device) Google 2 XL

sanchezzzhak committed:
feat (device) Huawei Honor 9X Pro (HLK-AL10)

sanchezzzhak committed:
feat (device) Hisense E Max (HLTE221E)

sanchezzzhak committed:
feat (device) Hisense H30 (HLTE223E)

sanchezzzhak committed:
feat (device) Huawei Honor 10 Lite (HRY-TL00), P20 Pro (HW-01K);

sanchezzzhak committed:
feat (device) HTC U11 (HTV33)

mimmi20 commented on regexes/device/mobiles.yml:
```diff
@@ -5173,7 +5173,7 @@ Huawei:
     - regex: 'COL-(?:AL[01]0|TL[01]0|L29)'
       model: 'Honor 10'
-    - regex: 'HRY-(?:AL00|LX[12])|HRY-LX1MEB'
+    - regex: 'HRY-(?:AL00|LX[12])|HRY-(LX1MEB|TL00)'
```
why dont you write this `- regex: 'HRY-(?:AL00|LX[12]|LX1MEB|TL00)'`?

sanchezzzhak replied:
Yes, I will do so. Done;

sanchezzzhak committed:
pr optimisation regexp detect huawei honor 10 lite

sanchezzzhak committed:
feat (device) Huawei Honor 8S (KSA-AL00, KSA-LX2);
feat (device) Haier I6 Infinity;

sanchezzzhak committed:
feat (device) Huawei Honor 20 Lite (LRA-AL00);

sanchezzzhak committed:
feat (device) LG Premier Pro LTE (LML413DL);

sanchezzzhak committed:
feat (device) Huawei P30 Lite (MAR-LX1B, MAR-TL00)

sanchezzzhak committed:
feat (device) ZTE Nubia Red Magic Mars (NX619J)
feat (device) ZTE Nubia Z9 mini
feat (device) Nubia Z9 Max
feat (device) Nubia Z20 (NX627J)
feat (device) Nubia Red Magic 3 (NX629J)

sanchezzzhak committed:
feat (device) T-Mobile REVVL Plus

sanchezzzhak committed:
feat (device) Sharp One S3 (S3-SH)

sanchezzzhak committed:
feat (device) Kyocera One S4 (S4-KC)

sanchezzzhak committed:
feat (device) Vivo X21S (V1814A)

sanchezzzhak committed:
feat (device) Lenovo Vibe K5 Plus

sanchezzzhak committed:
feat (device) Allview Viva 1003G

sanchezzzhak committed:
feat (device) Huawei MediaPad M6 8.4 (VRD-AL10)

sanchezzzhak committed:
feat (device) Huawei Honor 20S (YAL-AL50)

sanchezzzhak committed:
feat (device) Yandex smartphone (YNDX000SB)

sanchezzzhak committed:
feat (device) ViewSonic A8 Plus

sanchezzzhak committed:
feat (device) new brand Yxtel && detect model U3, U1 H1

sanchezzzhak committed:
feat (device) Ginzzu GT-7210, GT-7115

sanchezzzhak committed:
feat (device) Proline H10882M-TN;
feat (device) Haier L56 HM-N501-FL;
feat (device) Xiaolajiao Red Pepper Note 3

sanchezzzhak committed:
feat (device) fix Umax VisionBook P55 LTE Pro change type to smartphone;
feat (device) detect Umax Visionbook 7Q Plus, VisionBook 8Q LTE, VisionBook P50 LTE, VisionBook P55 LTE, VisionBook 8Q Plus

sanchezzzhak committed:
feat (device) Vivax Point X551

sanchezzzhak committed:
feat (device) Allview Viva 1001G

sanchezzzhak committed:
feat (device) Zopo Color S (ZP370)
feat (device) Zopo C2 II (ZOPO_C2_MOD)
feat (device) Zopo Color X (ZP563, ZP586)
feat (device) Zopo Color C5i (ZP567)
feat (device) Zopo Flash G5 Plus (ZP781)
feat (device) Zopo Speed 8 (ZP955)

sanchezzzhak committed:
feat (device) GOCLEVER INSIGNIA devices

sanchezzzhak committed:
feat (device) Toshiba Excite (AT7-C);
feat (device) fix Ark Benefit regexp;

sanchezzzhak committed:
feat (device) Assistant AP-109
feat (device) Casper VIA V8, VIA V5
feat (device) Huawei Honor Water Play 10.1 (HDN-W09)

sanchezzzhak committed:
feat (device) Yota Phone 2 (YD201)

sanchezzzhak committed:
feat (device) Casper VIA V3, VIA 10
feat (device) Huawei Honor Water Play 10.1 (HDN-W09)
feat (device) Lenovo Yoga Tab 2 Pro, Yoga Tab 2

sanchezzzhak committed:
feat (device) Panasonic Lumix DMC-CM1

sanchezzzhak committed:
feat (device) Mystery (MID-713G, MID-733G, MID-743G, MID-783G, MID-833G, MID-703G, MID-721, MID-722, MID-753G, MID-123G)

sanchezzzhak committed:
feat (device) Ainol Novo 10 Hero QuadCore

sanchezzzhak committed:
feat(device) detect new brand DeWalt model MD501
feat(device) detect Ainol Numy 3G Talos
feat(device) detect Casper VIA A1 Plus, VIA F1
feat(device) detect Vodafone Tab Prime 6
feat(device) detect Rombica Infinity K8
feat(device) detect LeEco X910
feat(device) detect Vestel Venus Go
feat(device) detect Assistant AS-502
feat(device) detect new brand Ritzviva model S500C

sanchezzzhak committed:
feat(device) detect new brand Runbo models: X5-W, F1 Plus, F1, TT, Q5-S
feat(device) detect Runbo models: F1-EN, F1-TT, X5-King
